package.json

```diff
@@ -34,7 +34,7 @@
     "watch:mobile:production": "NODE_ENV=mobile:production npm run commons:watch -- --env.target=mobile",
     "serve:mobile": "(cd src/targets/mobile/www && http-server -p 8005)",
     "mobile:icon": "(cd src/targets/mobile && splashicon-generator --imagespath='./res/model')",
-    "mobile:prepare": "(cd src/targets/mobile && cordova prepare && bundle install) && if [ $(uname) == 'Darwin' ] ; then npm run ios:install_pods ; fi",
+    "mobile:prepare": "(cd src/targets/mobile && cordova prepare && bundle install) && if [[ $(uname) == 'Darwin' ]] ; then npm run ios:install_pods ; fi",
```

Comment:
I switched to single brackets in https://github.com/cozy/cozy-banks/pull/1096/commits/bfe3b3143c3bf319358e720fc13a9326be464a6e because I had an error when using double brackets for a long time. What are the advantages of double brackets? Are they required?

Reply:
I always use [[.

> [[ has fewer surprises and is generally safer to use. But it is not portable

https://stackoverflow.com/questions/669452/is-double-square-brackets-preferable-over-single-square-brackets-in-ba

> Parentheses in [[ do not need to be escaped: 

> No WordSplitting or glob expansion will be done for [[ (and therefore many arguments need not be quoted): 

http://mywiki.wooledge.org/BashFAQ/031

Reply:
You can config yarn shell : 
```
npm config set script-shell bash
```

Reply:
> I switched to single brackets in bfe3b31 because I had an error when using double brackets for a long time.

Double brackets are only supported in bash, I am not sure we should oblige users of banks to configure npm with bash.